apps/rewards/contracts/Rewards.sol

```diff
@@ -216,7 +216,7 @@ contract Rewards is AragonApp {
         reward.delay = _delay;
         reward.blockStart = _startBlock;
         reward.creator = msg.sender;
-        emit RewardAdded(rewardId);
+        emit RewardAdded(rewardId, tx.origin);
```

Comment:
Same here: I am not really sure if this should be `msg.sender` instead... I invoke @Quazia for this or try to read this to ensure `tx.origin` is what we need: https://ethereum.stackexchange.com/questions/76929/tx-origin-alternative